src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java

```diff
@@ -189,7 +189,7 @@ public void should_return_a_200_and_retrieve_a_task_by_event_and_case_match() {
             CamundaVariableDefinition.JURISDICTION, "IA",
             CamundaVariableDefinition.LOCATION, "765324",
             CamundaVariableDefinition.TASK_ID, "reviewTheAppeal",
-            CamundaVariableDefinition.TASK_TYPE, "reviewTheAppeal",
+            CamundaVariableDefinition.TASK_TYPE, "Review The Appeal",
```

Comment:
This might be a bit confusing but the task type is the same as the taskId did you mean task_name ? 
```suggestion
            CamundaVariableDefinition.TASK_TYPE, "reviewTheAppeal",
            CamundaVariableDefinition.TASK_NAME, "Review The Appeal",
```

Reply:
task name removed